### PR TITLE
feat: Desactivate 2FA

### PR DESCRIPTION
**Pull request by Narkfr**

### CODE REVIEW

**Comment by Narkfr** [comment]
[(3) ETU je peux désactiver ma double authentification](https://trello.com/c/hZTjxpIC/841-3-etu-je-peux-d%C3%A9sactiver-ma-double-authentification)

File: src/styles/view.styl [hunk 1]
```diff
@@ -32,8 +32,12 @@
 
 .set-view-content-twofa
     &-modal
+      max-width 35rem !important
```
Comment (gregorylegarec) [inline src/styles/view.styl]: We must not use `!important` (cc @GoOz), could you be more specific in your selector ?

Comment (Narkfr) [reply src/styles/view.styl]: Done !

**Comment by gregorylegarec** [comment]
Thanks @Narkfr ⚙️

**Pull request merged**